<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPeer</class>
 <widget class="QDialog" name="DlgPeer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QLabel" name="lbl_env_status">
     <property name="text">
      <string>Status</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <layout class="QVBoxLayout" name="env_status"/>
   </item>
   <item row="3" column="1">
    <layout class="QVBoxLayout" name="env_name"/>
   </item>
   <item row="2" column="1">
    <widget class="Line" name="line_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QGroupBox" name="gr_ssh">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>IP</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="le_pass"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="le_user"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Pass</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_port"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_ip"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>User</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btn_ssh_peer">
        <property name="text">
         <string>SSH into Peer</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lbl_connection_status">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lbl_env">
     <property name="text">
      <string>Environment</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="btn_peer_on_hub">
     <property name="text">
      <string>Peer on Bazaar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lbl_env_owner">
     <property name="text">
      <string>Owner</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="btn_launch_console">
     <property name="text">
      <string>Launch  Subutai Console</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" alignment="Qt::AlignHCenter">
    <widget class="QCheckBox" name="show_ssh">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Show SSH box</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <layout class="QVBoxLayout" name="env_owner"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
